src/super_gradients/training/models/detection_models/yolo_base.py

```diff
@@ -80,11 +80,14 @@ def __init__(
         self.with_confidence = with_confidence
 
     def forward(self, x, device: str = None):
+        # Use the main output features in case of multiple outputs.
+        if isinstance(x, (tuple, list)):
```

Comment:
Mind adding a unit test to ensure that this change passes two sanity checks? 

postprocessing_callback(model.eval().forward(dummy_input))
postprocessing_callback(model.train().forward(dummy_input))

Reply:
model should be one of the detection models, i.e yolox? Or just a toy model? @BloodAxe

Reply:
YoloX. Thanks

Reply:
Sorry for the late reply, added [post prediction unit test](https://github.com/Deci-AI/super-gradients/pull/813/commits/24ac09841750118dac77dd5c54ddc066764ac07b)

Reply:
`postprocessing_callback(model.train().forward(dummy_input))` will fails even without this PR since in train mode we don't decode bounding box (using DetectX module), and only logits are outputted